bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/api/Car.java

```diff
@@ -105,6 +114,9 @@ public void setBatteryStatus(JsonObject responseJson) {
                 if (attributes.get("chargingRemainingTime") != null) {
                     chargingRemainingTime = attributes.get("chargingRemainingTime").getAsInt();
                 }
+                if (attributes.get("timestamp") != null) {
+                    batteryStatusUpdated = attributes.get("timestamp").getAsString();
```

Comment:
Which format does this timestamp string have? Perhaps it would be better to convert it to an `Instant` or `ZonedDateTime` here, so callers of `getBatteryStatusUpdated` won't have to know the format and deal with it.